lib/spotify_api.rb

```diff
@@ -55,10 +55,92 @@ def find_song_spotify song
         :album_name =>  song["album"]["name"],
         :album_image => song["album"]["images"][2],
         :artist => song["artists"][0]["name"],
-        :preview_url => song["preview_url"]
+        :preview_url => song["preview_url"],
+        :uri => song["uri"]
       }
       song_dets.push(finds)
     end
     song_dets
   end
+
+
+  def create_playlist_spotify
+    r =  refresh_if_needed do
+     HTTParty.post "https://api.spotify.com/v1/users/tiydc/playlists",
+      headers:{
+        "Accept" => "application/json",
+        "Authorization" => "Bearer #{key}",
```

Comment:
Another thought for later: there's a little bit of duplication here (in having to know where Spotify's API is and how to authenticate). It might make sense to pull out a class or method that knows about that. See [e.g. this](https://github.com/jnunemaker/httparty/blob/master/examples/twitter.rb) for an idea about how you could do that.